code/game/objects/structures/watercloset.dm

```diff
@@ -467,6 +467,8 @@
 	desc = "A sink frame, that needs a water recycler to finish construction."
 	anchored = FALSE
 	material_flags = MATERIAL_ADD_PREFIX | MATERIAL_COLOR | MATERIAL_AFFECT_STATISTICS
+	var/buildstacktype = null
+	var/buildstackamount = 1
 
```

Comment:
Why have these exist if they're unused?

Reply:
I initially used the normal sink deconstruction as a reference and had this set to iron, but it caused every sink to spit out iron regardless of what material it was made of. I probably should've tested whether they were necessary at all rather than just setting it to null

Reply:
:)